cmd/ztest/Makefile.am

```diff
@@ -2,7 +2,7 @@ include $(top_srcdir)/config/Rules.am
 
 # -Wnoformat-truncation to get rid of compiler warning for unchecked
 #  truncating snprintfs on gcc 7.1.1.
-AM_CFLAGS += $(DEBUG_STACKFLAGS) $(FRAME_LARGER_THAN) $(NO_FORMAT_TRUNCATION)
+AM_CFLAGS += $(DEBUG_STACKFLAGS) $(FRAME_LARGER_THAN) $(NO_FORMAT_TRUNCATION) $(COMPILER_FSANITIZE_ADDRESS)
```

Comment:
Let's pull `$(COMPILER_FSANITIZE_ADDRESS)` down on to it's own line to mind the 80 character limit.  Here and elsewhere.

```
AM_CFLAGS += $(DEBUG_STACKFLAGS) $(FRAME_LARGER_THAN) $(NO_FORMAT_TRUNCATION)
AM_CFLAGS += $(COMPILER_FSANITIZE_ADDRESS)
```